scipy/special/tests/test_basic.py

```diff
@@ -4060,34 +4060,69 @@ class TestStirling2:
     def test_table_cases(self):
         for n in range(len(self.table)):
             for k in range(len(self.table[n])):
-                assert_equal(self.table[n][k], stirling2(n, k))
+                assert_equal(self.table[n][k], stirling2(n, k, exact=True))
+                assert_allclose(self.table[n][k],
+                                stirling2(n, k, exact=False),
+                                rtol=1e-12,
+                                )
```

Comment:
Why not  `@pytest.mark.parametrize("is_exact", [True, False])`?

Reply:
no particular reason, I can restructure and swap out the comparators as well